Don't try to get PIC flags from libtool, because libtool has not been generated
for the current platform by this point. The libtool that gets run is the bundled
one which was generated on Linux, which doesn't work on macOS which results in
errors being displayed during configuration:

./libtool: line 909: /bin/sed: No such file or directory
./libtool: line 915: /bin/sed: No such file or directory
./libtool: line 2640: /bin/sed: No such file or directory

https://github.com/oetiker/rrdtool-1.x/issues/898
https://github.com/oetiker/rrdtool-1.x/commit/b3f6be4f43db57c68c3262ac373f115b3c861ae7#diff-67e997bcfdac55191033d57a16d1408a
--- configure.orig	2017-05-16 06:55:23.000000000 -0500
+++ configure	2018-08-07 18:26:28.000000000 -0500
@@ -15046,8 +15046,6 @@
 
 
 
-eval `./libtool --config | grep pic_flag=`
-CFLAGS="$CFLAGS $pic_flag"
 
 
 for ac_func in strftime
